internal/plugins/ansible/constants/constants.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2018 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package constants
+
+import (
+	"path/filepath"
+)
+
+const (
+	FilePathSep          = string(filepath.Separator)
+	RolesDir             = "roles"
+	MoleculeDir          = "molecule"
+	MoleculeDefaultDir   = MoleculeDir + FilePathSep + "default"
+	MoleculeTestLocalDir = MoleculeDir + FilePathSep + "test-local"
+	MoleculeClusterDir   = MoleculeDir + FilePathSep + "cluster"
+	MoleculeTemplatesDir = MoleculeDir + FilePathSep + "templates"
+)
+
+// TODO(asmacdo) can this go?
+// AnsibleDelims is a slice of two strings representing the left and right delimiters for ansible templates.
+// Arrays can't be constants but this should be a constant.
+var AnsibleDelims = [2]string{"[[", "]]"}
```

Comment:
IMO this file is not required see that each const is used just once in the boilerplate that will use that to scaffold the file,